src/core/statistics/useStatistics.ts

```diff
@@ -0,0 +1,24 @@
+export function useStatistics() {
+  // If the global wts object doesn't exist, it means we are in dev environment.
+  // Here instead of actually tracking we just log the data to the console.
+  if (!window.wts) {
+    window.wts = {
+      push(trackingProps: [string, string, { [key: string]: string }]) {
+        // eslint-disable-next-line no-console
+        console.log(
+          `Tracking: ${trackingProps[0]}, ${trackingProps[1]}, ${JSON.stringify(
+            trackingProps[2]
+          )}`
+        );
+      }
+    };
+  }
+
+  return {
+    track: (trackedData: { [key: string]: string }) => {
+      window.wts.push(["send", "page", trackedData]);
+    }
```

Comment:
Consider returning multiple functions with different signatures supporting different behavior.

There may be different behavior and specifications for page/update and click/link tracking.

Reply:
I adjusted this, but not into returning multiple functions, because I believe we can achieve all we want with the setup. Let's see if it if it works.